source/blog.html.haml

```diff
@@ -0,0 +1,32 @@
+---
+title: Blog
+pageable: true
+per_page: 5
+---
+
+%h2.blog_heading
```

Comment:
Don't use underscores like this in class names. Use hyphens instead. Exception to that would be double underscores for BEM style classes. Syntax example from [this BEM article](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) looks like this:

```css
.site-search {} /* Block */
.site-search__field {} /* Element */
.site-search--full {} /* Modifier */
```